### PR TITLE
FUFB-246 PDP: Variant options, QTY selector, add to bag

### PR DESCRIPTION
**Pull request by srdan89**

### CODE REVIEW

File: sites/_shared/assets/global.js [hunk 1]
```diff
@@ -741,7 +741,7 @@ class VariantSelects extends HTMLElement {
     this.setColorVariantLabel();
     this.setPriceLabelAddButton();
     this.updateVariantMedia();
-    this.addOutOfStockClass();
+    //this.addOutOfStockClass();
```
Comment (lgilders) [inline sites/_shared/assets/global.js:745]: Let's remove commented out code.

File: sites/_shared/assets/global.js [hunk 2]
```diff
@@ -752,7 +752,7 @@ class VariantSelects extends HTMLElement {
     this.removeErrorMessage();
     this.setColorVariantLabel();
     this.removeColorVariantLabel();
-    this.addOutOfStockClass();
+    //this.addOutOfStockClass();
```
Comment (lgilders) [inline sites/_shared/assets/global.js:758]: Let's remove commented out code.

**Review by lgilders — APPROVED** [review]
PR approved; no changes needed.

File: sites/_shared/assets/global.js (hunk 1 above)
Comment (srdan89) [reply sites/_shared/assets/global.js:745]: it is removed on the outofstock ticket

**Review by lgilders — APPROVED** [review]
PR reviewed; no changes needed.

**Pull request merged**